sms2jwplayer/csv.py

```diff
@@ -58,3 +151,5 @@ def load(fobj, skip_header_row=True):
         MediaItem._make([t(v) for t, v in zip(_MEDIA_ITEM_TYPES, row)])
         for row in reader
     ]
+
```

Comment:
We have a number of PEP8 violations in this code :S but let's not introduce a new one by adding blank lines.